docs/integrations/github/discovery.md

```diff
@@ -53,6 +53,67 @@ And then add the entity provider to your catalog builder:
   }
 ```
 
+## Installation with Events Support
+
+Please follow the installation instructions at
+
+- https://github.com/backstage/backstage/tree/master/plugins/events-backend/README.md
+- https://github.com/backstage/backstage/tree/master/plugins/events-backend-module-github/README.md
+
+Additionally, you need to decide how you want to receive events from external sources like
+
+- [via HTTP endpoint](https://github.com/backstage/backstage/tree/master/plugins/events-backend/README.md)
+- [via an AWS SQS queue](https://github.com/backstage/backstage/tree/master/plugins/events-backend-module-aws-sqs/README.md)
+
+Set up your provider
+
+```diff
+// packages/backend/src/plugins/catalogEventBasedProviders.ts
++import { GithubEntityProvider } from '@backstage/plugin-catalog-backend-module-github';
+ import { EntityProvider } from '@backstage/plugin-catalog-node';
+ import { EventSubscriber } from '@backstage/plugin-events-node';
+ import { PluginEnvironment } from '../types';
+ export default async function createCatalogEventBasedProviders(
+-  _: PluginEnvironment,
++  env: PluginEnvironment,
+ ): Promise<Array<EntityProvider & EventSubscriber>> {
+   const providers: Array<
+     (EntityProvider & EventSubscriber) | Array<EntityProvider & EventSubscriber>
+   > = [];
+-  // add your event-based entity providers here
++  providers.push(
++    GithubEntityProvider.fromConfig(env.config, {
++       logger: env.logger,
++       // optional: alternatively, use scheduler with schedule defined in app-config.yaml
++       schedule: env.scheduler.createScheduledTaskRunner({
++         frequency: { minutes: 30 },
++         timeout: { minutes: 3 },
++       }),
++       // optional: alternatively, use schedule
++       scheduler: env.scheduler,
++    }),
++  );
+   return providers.flat();
+ }
+```
+
+### Set up HTTP webhook
+
+Next, you will need to subscribe to a GitHub webhook. Navigate to your repository of choice (this should be a repository you own, oy you can configure for your organization too) and go to _Settings → Webhooks_.
+On the Webhooks page, click on the _Add webhook_ button on the top right-hand corner.
+
+On the webhook form displayed, paste the webhook URL, something like `YOUR_BACKSTAGE_URL/api/events/http/github`, into the Payload URL field.
+
+Fill the remaining fields as follows:
+
+- Content type: Select application/json so that you can receive the payload as a JSON object.
+- Secret: You can leave this blank
```

Comment:
This could be changed after https://github.com/backstage/backstage/pull/14689 be merged I believe